data/build.gradle.kts

```diff
@@ -16,6 +16,7 @@
 plugins {
     id("droidconke.android.library")
     id("com.google.devtools.ksp")
```

Comment:
You can now remove line 18, as the room plugin is providing ksp to this module

Reply:
On the same file, remove this block as well

```
 ksp {
        arg("room.schemaLocation", "$projectDir/schemas")
    }
```

Its also configured inside `AndroidRoomConventionPlugin`

Reply:
Thanks, I get it now.